tests/query/test_cli.py

```
@@ -2,8 +1,11 @@
+from sema.query.__main__ import _main as query_main
 
 
 def test_basic():
```

Ruff

Missing return type annotation for public function `test_basic`

Add return type annotation: `None`

(ANN201)